src/platform/audio/ffmpegaudiofilecache.cpp

```diff
@@ -134,7 +134,7 @@ bool FFmpegAudioFileCache::Open_FFmpeg_Contexts(FFmpegOpenAudioFile *open_audio,
         return false;
     }
 
-    AVCodec *input_codec = avcodec_find_decoder(open_audio->fmt_ctx->streams[0]->codecpar->codec_id);
+    const AVCodec *input_codec = avcodec_find_decoder(open_audio->fmt_ctx->streams[0]->codecpar->codec_id);
```

Comment:
Why is this here?

Reply:
Well i accidentally committed this since i need it locally to compile. However the diff should disappear once #780 is merged